pkgs/racket-doc/scribblings/reference/vectors.scrbl

```diff
@@ -47,7 +47,20 @@ all elements, so for mutable data, mutating an element will affect other element
   v
 ]
 
-This function takes time proportional to @racket[size].}
+This function takes time proportional to @racket[size].
+
+A common mistake is using @racket[make-vector] to create nested vectors.
+The following creates a vector that contain 3 vectors to the same vector.
+@examples[
+  (make-vector 3 (vector)) ; wrong
+]
+
+To create nested vectors, do this instead:
+@examples[
+(for/vector ([_ (in-range 0 n)])
+  (vector))
```

Comment:
Two things: 

* You used "3" in the first example, I would use it in the second one as well.
* I think it's probably slightly more idiomatic to omit the in-range, and just write
```
(for/vector ([_ 3])
  (vector))
```

... but I could be persuaded that I'm wrong.

Reply:
Ah, I didn't know that, I just copied what jack said on discord. It's better not to use `in-range` since it's shorter

Reply:
I actually prefer the explicit `in-range`, personally. I think the way `for` loops treat bare numbers is unintuitive. But Resyntax has no official stance on the issue.